packages/api-client/src/views/Request/ResponseSection/ResponseHeaders.vue

```diff
@@ -26,7 +26,7 @@ const findHeaderInfo = (name: string) => {
     <template #title>Headers</template>
     <div
       v-if="headers.length"
-      class="border-t-1/2 border-b-1/20 max-h-[calc(100%-32px)] overflow-y-auto">
+      class="border-t-1/2 border-b-1/2 max-h-[calc(100%-32px)] overflow-y-auto">
```

Comment:
I think this can just be border-y (or at least it doesn't need the 1/2)

Reply:
yes! do you mean `border-b`?

Reply:
prob a couple in the app to 🧹

Reply:
I think `border-y` should apply border-b and border-t for you

Reply:
oooh didn't see that border-t 😄 great catch